lang/de.json

```diff
@@ -14,19 +14,20 @@
   "/LjEZJ": "Contacted <Account></Account>",
   "/rbxQW": "Activated <Account></Account> as a host",
   "03Q893": "Dies kann später bearbeitet werden",
-  "05doZ2": "Zurück zu all Ihren Apps",
-  "07Y/8I": "Fortfahren mit Ihrem OpenCollective Konto",
-  "0JzeTD": "{taxName} anwenden",
-  "0vL5u1": "Konto erstellen",
-  "104ECN": "Private Spende - Siehe Privatsphäre-Feld für Informationen",
-  "10vwJU": "Dieses Konto ist zurzeit gesperrt und kann nicht verwendet werden, um Ereignisse zu erstellen.",
+  "05doZ2": "Go back to all your apps",
+  "07Y/8I": "Continue with your Open Collective account",
+  "0JzeTD": "Apply {taxName}",
+  "0vL5u1": "Create an account",
+  "104ECN": "Private contribution - Check privacy box for info",
+  "108gPp": "Learn more about tiers",
+  "10vwJU": "This account is currently frozen and cannot be used to create events.",
   "1cWx1w": "Communities around the world are using Open Collective. Find out more about them!",
-  "1EYCdR": "Schlagwörter",
-  "1gjV2o": "Umsatzsteuer-Identifikationsnummer",
-  "1KQrEf": "Noch kein Konto?",
-  "1lIftz": "Sie können Apps erstellen, die sich mit der Open Collective Plattform verbinden. <CreateAppLink>Erstellen Sie eine App</CreateAppLink> mit der Open Collective API.",
-  "1rESHf": "Für Sponsoren",
-  "1Whmi8": "Dieses Kollektiv (und alle damit verbundenen Projekte und Veranstaltungen) hat nun die Möglichkeit, Gelder anzunehmen, Ausgaben zu tätigen, Updates zu veröffentlichen und neue Veranstaltungen oder Projekte zu erstellen.",
+  "1EYCdR": "Tags",
+  "1gjV2o": "VAT identifier",
+  "1KQrEf": "Don't have one?",
+  "1lIftz": "You can create apps that integrate with the Open Collective platform. <CreateAppLink>Create an app</CreateAppLink> using the Open Collective's API.",
+  "1rESHf": "For Sponsors",
+  "1Whmi8": "This collective (and all its related Projects & Events) will now have access to accept funds, pay out expenses, post updates, create new Events or Projects.",
```

Comment:
Something went wrong here, probably while merging `main`; we don't want to remove the existing translations. We usually recommend rebasing rather than merging `main`, it produces cleaner histories and these kinds of conflicts are usually easier to manage.

We should checkout the files from `main`, then run `npm run build:langs` to make sure everything is in sync.